Sources/Orbit/Components/Stepper.swift

```diff
@@ -0,0 +1,284 @@
+import SwiftUI
+
+/// Enables incremental changes of a counter without a direct input.
+///
+/// - Note: [Orbit definition](https://orbit.kiwi/components/interaction/stepper/)
+public struct Stepper: View {
+    
+    @Binding var value: Int
+    
+    let minValue: Int
+    let maxValue: Int
+    let style: Style
+    
+    @Environment(\.isEnabled) var isEnabled
+    
+    public var body: some View {
+        HStack(alignment: .center, spacing: 0) {
+            decrementButton
+            
+            valueText
+                .frame(minWidth: 40)
+            
+            incrementButton
+        }
+    }
+    
+    @ViewBuilder var valueText: some View {
+        Text("\(value)")
+            .accessibility(.stepperValue)
+    }
+    
+    @ViewBuilder var decrementButton: some View {
+        StepperButton(
+            isIncrement: false,
+            isEnabled: isEnabled ? value > minValue : false,
+            style: style,
+            action: {
+                value -= 1
+            }
+        )
+        .accessibility(.stepperDecrement)
+    }
+    
+    @ViewBuilder var incrementButton: some View {
+        StepperButton(
+            isIncrement: true,
+            isEnabled: isEnabled ? value < maxValue : false,
+            style: style,
+            action: {
+                value += 1
+            }
+        )
+        .accessibility(.stepperIncrement)
+    }
+    
+    /// Creates Orbit Stepper component.
+    public init(
+        value: Binding<Int>,
+        minValue: Int,
+        maxValue: Int,
+        style: Style
+    ) {
+        self._value = value
+        self.minValue = minValue
+        self.maxValue = maxValue
+        self.style = style
+    }
+}
+
+struct StepperButton: View {
```

Comment:
The `StepperButton` can be extracted in its own file in a `support/components` folder, where we store all these supporting components. It should also be `public` so a call site can use it as a standalone component.

Reply:
Extracted 👍🏻